samples/highcharts/demo/combo-data-resolution/demo.html

```diff
@@ -0,0 +1,316 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/export-data.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<figure class="highcharts-figure">
+    <div id="container"></div>
+    <p class="highcharts-description">
+        Chart showing a combination of a column and a line chart across time,
+        each series representing a shared dataset but at different resolutions.
+    </p>
+</figure>
+
+<pre id="columnData">
+    [
+        [
+            1451520000000,
+            0.9363
+        ],
+        [
+            504835200000,
+            0.2296
+        ],
+        [
+            -441936000000,
+            -0.199
+        ],
+        [
+            -1388620800000,
+            -0.0667
+        ],
+        [
+            -2335305600000,
+            -0.0974
+        ]
+    ]
+</pre>
+
+<pre id="lineData">
+    [
```

Comment:
This data is sorted descendingly, causing an Highcharts warning 12. Also, it seems to be repeating two by two values.